lib/strMap.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference path='../typings/tsd.d.ts' />
+
+export = StrMap;
```

Comment:
It's a util class that Tzvetan wrote and I found it quite useful. I did not know a ES6 map is coming, and would agree with you to not having a separate implementation. I will change it to a raw `Object`.